src/featurs/cartSlice.ts

```diff
@@ -0,0 +1,25 @@
+export const a = null;
+// import { createSlice, PayloadAction } from "@reduxjs/toolkit";
```

Comment:
what is this file use for.

Reply:
pending I will work on this file tomorrow. it will hold the card state.